src/components/controllers/page/index.js

```diff
@@ -157,8 +45,9 @@ export default function Page({ route, prev, next }, ctx) {
 	// "name" is the exact page ID from the URL
 	// "current" is the currently *displayed* page ID.
 
-	const showTitle = current != 'index' && meta.show_title !== false;
-	const canEdit = showTitle && current != '404' && current !== '/blog';
+	const isHome = current === 'index';
+	const showTitle = meta.show_title !== false;
+	const canEdit = !isHome;
```

Comment:
`showTitle`, as far as I can tell, is only used to differentiate the home page from the rest of the content. The problem I had with this is that I thought the spacing on every other page was rather nice, as it kept the content from bunching up against the header on pages where we didn't have a title, such as the blog:

With Spacer             |  Without Spacer
:-------------------------:|:-------------------------:
![With spacer](https://github.com/preactjs/preact-www/assets/33403762/4d1952cf-0e21-48da-ae4d-413a3e8bf46d) |  ![Without spacer](https://github.com/preactjs/preact-www/assets/33403762/f94e5cdb-4a84-4f2a-9a1c-7f210493bb13)

As such, I split this up into `isHome` and `showTitle`. As for `canEdit`, I tend to that's fine to show on every page besides home? Happy to revert though.

Coincidentally, on `/blog` the title wasn't supposed to be shown at all but `current` is actually `blog`, not `/blog`. I kinda prefer it though?